wemake_python_styleguide/options/defaults.py

```diff
@@ -120,3 +120,6 @@
 
 #: Maximum number of nested annotations.
 MAX_ANN_COMPLEXITY: Final = 3
+
+#: Maximum number of names that can be imported from module.
+MAX_IMPORT_FROM_MEMBERS: Final = 8  # guessed
```

Comment:
We have set this to `8` by default, by our own code has `20`

What do you think? Should we adjust this value? Or refactor our own code?
I am asking because `8` looks great. But, linter has found new problem in our code that we might fix. Or just ignore.

Reply:
I have checked some of my personal and work projects and 8 is usually the maximum number of imports from one module.
I have set this value for our project to such big number, because it's library, however now I double checked and there is only 7 `from ... import ...` that import more than 8 members and they are related to:

+ violations - so we can just import namespace and use namespaced violations (imho it's even clearer)
+ ast - same story as with violations
+ typing - again same story, but if you prefer to import everything explicitly from `typing` we can add some option to pass module names that can be skipped for this violation
+ visitors imports in presets - same story like with `typing` imports

What's your opinion?

Reply:
- violations: I agree, let's do this
- ast: agreed
- typing: yes, I feel like this is a good idea. But quite problematic. One can have a custom module named `typing`, `typing` might have submodules like `.re` or `.decorators`. So we will have to add a lot of configuration for this. I don't like configuration. I guess we can just leave `noqa` comment for this case
- visitors: `noqa` looks like the best option